From f58f3b1d71adf1686bda60e15cececa696154d46 Mon Sep 17 00:00:00 2001
From: Marcin Niestroj <m.niestroj@emb.dev>
Date: Sat, 11 Nov 2023 16:28:23 +0100
Subject: [PATCH] samples: http_get: increase mbedTLS heap size

40k heap size is not enough and results in handshake errors:

  <err> net_sock_tls: TLS handshake error: -0x2700

Increasing it to 55k already works (when used with native_sim_64 and
offloaded sockets), but use 60k to get a bit more room for future.

Signed-off-by: Marcin Niestroj <m.niestroj@emb.dev>
---
 samples/net/sockets/http_get/overlay-tls.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/samples/net/sockets/http_get/overlay-tls.conf b/samples/net/sockets/http_get/overlay-tls.conf
index 940d7f018cf..3dd9096519f 100644
--- a/samples/net/sockets/http_get/overlay-tls.conf
+++ b/samples/net/sockets/http_get/overlay-tls.conf
@@ -4,7 +4,7 @@ CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_MBEDTLS=y
 CONFIG_MBEDTLS_BUILTIN=y
 CONFIG_MBEDTLS_ENABLE_HEAP=y
-CONFIG_MBEDTLS_HEAP_SIZE=40000
+CONFIG_MBEDTLS_HEAP_SIZE=60000
 CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=7168
 
 CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
-- 
2.43.2

